include/boost/url/url_base.hpp

```diff
@@ -1986,6 +2001,34 @@ class url_base::params_type
     string_type<Allocator>
     at( string_view key,
         Allocator const& a = {}) const;
+
+    BOOST_URL_DECL
+    iterator
+    erase( iterator b, iterator e ) noexcept;
```

Comment:
The canonical formal parameter names are `first` and `last`. Yes I realize that "last" is a bit misleading but this is better than "b & e".